traffic_ops/traffic_ops_golang/origin/origins.go

```diff
@@ -487,3 +488,277 @@ func deleteQuery() string {
 WHERE id=:id`
 	return query
 }
+
+// Get is the handler for GET requests to Origins of APIv5.
+func Get(w http.ResponseWriter, r *http.Request) {
+
+	var useIMS bool
+
+	inf, sysErr, userErr, errCode := api.NewInfo(r, nil, nil)
+	if sysErr != nil || userErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	origins, userErr, sysErr, errCode, _ := getOrigins(w.Header(), inf.Params, inf.Tx, inf.User, useIMS)
+	var returnable []tc.OriginV5
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+
+	for _, origin := range origins {
+		returnable = append(returnable, origin.ToOriginV5())
+	}
+
+	// Assign an empty array if no origin is found instead of null.
+	if len(returnable) == 0 {
+		returnable = []tc.OriginV5{}
+	}
```

Comment:
You don't need to do this if instead you just never set it to an incorrect value; the response can be initialized directly as a zero-length slice immediately.

Reply:
removed this line of code as you don't need to do anything here if there are no origins, the response will be an empty array by default